app/lib/db_utils.py

```diff
@@ -54,7 +54,7 @@ def update_object(data, obj_type, obj_id):
 
     if obj:
         for attr, value in data.items():
-            if type(value) == dict:
+            if isinstance(value) == dict:
```

Comment:
I believe this was fixed in a later version of develop, and the correct way to do it would be `isinstance(value, dict)`